packages/e2e-tests/.gitignore

```diff
@@ -0,0 +1 @@
+.env
```

Comment:
l: should we add an `.env.example`?

Reply:
👍 8561e3040b35cce5ba5a3863f5a0b3c7bde17198